mlflow/server/js/package.json

```diff
@@ -17,6 +17,7 @@
     "json-bigint": "databricks/json-bigint#a1defaf9cd8dd749f0fd4d5f83a22cd846789658",
     "leaflet": "^1.5.1",
     "lodash": "^4.17.11",
+    "lodash.debounce": "^4.0.8",
```

Comment:
Do we need a new package for this? Will `_.debouce` do the same?

Reply:
Generally speaking, `_.debounce` will do the same. However, as far as I know, lodash itself can be pretty heavy. That's why I imported a specific module instead of importing the whole lodash library. As far as I know `import _ from 'lodash'` will not tree-shake. I am not 100% sure about the newer versions, so please let me know if my assumptions are not correct. Would you like me to use `_.debounce` instead?